pkg/cmd/controller/route-controller/network.go

```diff
@@ -8,22 +8,22 @@ import (
 	"github.com/openshift/openshift-controller-manager/pkg/route/ingressip"
 )
 
-func RunIngressIPController(ctx *ControllerContext) (bool, error) {
```

Comment:
this should be the same. just take the routeControllerContext

Reply:
We're actually taking `OpenshiftControllerConfig.Ingress.IngressIPNetworkCIDR` here